### ISSUE TITLE
Simple button to display a number for one or more Simulation Variables

### ISSUE THREAD

**Issue by sromeroi** [nguyenquyhy/Flight-Tracker-StreamDeck#352]
**Is your feature request related to a problem? Please describe.**
I have Simple Gauge buttons for Airspeed, Throttle RPM, Trim value, Altitude, Vertical Speed, etc. For some of them is nice to have the "gauge" but for other, I woud like just to have the value and the title, like:

```
  5500
   ALT
```

**Describe the solution you'd like or alternatives you've considered**

Different ways to achieve this:

1.- Take advantage of the "gauge button" and add an option for "Hide gauge".

2.- Add a "Font Style" option for Display Value in "Toggle buttons" (I can use the Feedback Value to read any variable like INDICATED_ALTITUDE but the number is quite small).

3.- Create a new "Simple Value" button with Title and Simvar properties. It could even allow to read/display multiple values, like:

```
SIMVAR1(user presses enter here)
SIMVAR2
```

And that would display as:

```
1234
3452
```

Or with 2-3 "SimVar" boxes in the button.

Ideally, both options 1 and 3 would be perfect.

**Comment by sromeroi** [comment]
![Photo 2-9-22, 13 09 03](https://user-images.githubusercontent.com/4025149/188257292-9f25217a-f968-4c66-a1db-5fdc546102e6.jpg)

Picture of my current "six-pack screen". I'm trying to remove the gauge from ALT, HDG, VS and TRIM and also provide them the capability to trigger "events" (toggle something) or set simVar values on press/hold.

**Comment by rmroc451** [comment]
Pretty sure instead of using gauge button,  you can just use https://github.com/nguyenquyhy/Flight-Tracker-StreamDeck/blob/main/docs/USERGUIDE.md#generic-toggle-button

Give that a try?

**Comment by sromeroi** [comment]
> Pretty sure instead of using gauge button, you can just use https://github.com/nguyenquyhy/Flight-Tracker-StreamDeck/blob/main/docs/USERGUIDE.md#generic-toggle-button

> 
> Give that a try?

Yes, that was Option 2.- in my original request (reuse Generic Toggle by adding font style and position options for the Display Value item:

![Photo 6-9-22, 16 16 07](https://user-images.githubusercontent.com/4025149/188659057-93ac4cb9-1ec9-4d25-a18a-ba82ffeab494.jpg)

The problem is that the number is too small compared with the Gauge values:

![Photo 6-9-22, 16 15 42](https://user-images.githubusercontent.com/4025149/188659063-6b3dd41d-46da-472a-95a1-6735a3276a6d.jpg)

**Comment by sromeroi** [comment]
> I guess at some point I should merge those 2 button types as a lot should be common between the 2. 
> Maybe the generic button can have 3 different display mode: no gauge, simple gauge and custom gauge.

Good solution.

> For now, maybe I'll add in font size for both buttons. I do see issues occasionally when the number are too long/short.

perfect, thanks!

> And I am not sure what you mean by this `Add the Trigger Events/Set values
> on press/hold explained in 1.-` in option 2. I thought it is already there.

Yes, that's right, they are present in option 2. It is missing in the Gauge buttons, and it would be very useful to be able to trigger 2 different events (on press and on hold, with their respective parameters) when "pressing" gauge buttons.

Thanks a lot for your quick response.

**Comment by nguyenquyhy** [comment]
I split this into #378 and #379 and close this one for now.

**Comment by sromeroi** [comment]
Thanks!
